lib/msal-angular/docs/v2-docs/redirects.md

```diff
@@ -180,6 +180,8 @@ export class AppComponent implements OnInit, OnDestroy {
 
 ## 2. Subscribing to `handleRedirectObservable` manually
 
+**Note:** As of MSAL v3.x, the `MsalRedirectComponent` is now **MANDATORY** and manual subscriptions to `handleRedirectObservable` can no longer be guaranteed to work.
```

Comment:
Can we go with a stronger "are not supported" vs "no longer guaranteed to work"?

Reply:
I agree, I think we need to take a firmer stance one way or the other. I am curious though why `handleRedirectObservable` is no longer expected to work?

Reply:
As of this release, bootstrapping `MsalRedirectComponent` is the only way to properly initialize the application. Using `handleRedirectObservable` to manually handle redirects was an alternative to using `MsalRedirectComponent`. At point, using it directly as a redirect strategy (which is what this doc covers) will no longer work as the app will not be initialized properly.

Reply:
Is there a way someone could initialize & use handleRedirectObservable (initializationObservable perhaps?) I seem to remember quite a few people stating they could not or did not want to bootstrap the RedirectComponent

Reply:
Or maybe add initialize right before handleRedirectPromise on this line?

Reply:
The `MsalRedirectComponent` does not seem compatible with the new `bootstrapApplication` API from Angular as it only allows bootstrapping one component. Maybe it can be exposed as standalone component? Or should `handleRedirectObservable` still be supported?